mono/utils/mono-merp.c

```diff
@@ -206,95 +217,60 @@ parse_exception_type (const char *signal)
 	g_error ("Merp doesn't know how to handle %s\n", signal);
 }
 
-static gchar *
-mono_encode_merp_params (MERPStruct *merp)
-{
-	GString *output = g_string_new ("");
+#define FORMAT_TEXT_TO_FILE(handle, buff, sizeof_buff, ...) do { \
```

Comment:
`MOSTLY_ASYNC_FPRINTF`? So it more closely match the name `MOST_ASYNC_PRINTF`, and so you can even reimplement the latter in term of the former.